cli/dispatch_minimal.rs

```diff
@@ -133,7 +133,12 @@ mod ops {
 
   pub fn read(rid: i32, zero_copy: Option<PinnedBuf>) -> Box<MinimalOp> {
     debug!("read rid={}", rid);
-    let zero_copy = zero_copy.unwrap();
+    let zero_copy = match zero_copy {
+      None => {
+        return Box::new(futures::future::err(errors::no_buffer_specified()))
+      }
+      Some(buf) => buf,
+    };
```

Comment:
I think this could be simplified to:
```rust
if zero_copy.is_none() {
  return odd_future(errors::no_buffer_specified());
} 
```

Reply:
I think `unwrap()` is also required if `is_none()` is used. i.e.
```rust
if zero_copy.is_none() {
  return odd_future(errors::no_buffer_specified());
}
let zero_copy = zero_copy.unwrap();
```

IMHO, I prefer using `match` to `unwrap()`.